benchmarks/haskell/Benchmarks/FileWrite.hs

```diff
@@ -0,0 +1,33 @@
+-- | Benchmarks simple file writing
+--
+-- Tested in this benchmark:
+--
+-- * Writing a file to the disk
+--
+
+{-# LANGUAGE BangPatterns #-}
+
+module Benchmarks.FileWrite
+    ( benchmark
+    ) where
+
+import Test.Tasty.Bench (Benchmark, bgroup, bench, nfIO)
+import Data.Text (StrictText)
+import qualified Data.Text.IO as T
+import Data.Text.Lazy (LazyText)
+import qualified Data.Text.Lazy.IO as LT
+import System.IO (Handle)
+
+benchmark :: StrictText -> LazyText -> Handle -> Handle -> Handle -> Benchmark
+benchmark !st !lt !noBufH !lineBufH !blockBufH = bgroup "FileWrite"
+  [ bgroup "StrictText"
+    [ bench "NoBuffering" $ nfIO $ T.hPutStr noBufH st
```

Comment:
In general it's better to use `nfAppIO`, and I suggest we do so in new code.

Reply:
I think I got the text data to normal form but I am not  familiar with nfAppIO so please check. Thanks